executor/importer/import.go

```diff
@@ -127,6 +128,38 @@ type LoadDataReaderInfo struct {
 	Remote *mydump.SourceFileMeta
 }
 
+type Plan struct {
```

Comment:
this plan is used for serializing?

maybe make it a wrapped struct in controller, so no need duplicate fields

Reply:
I can add it later. Because all fields should be changed to public, the change will cause this pr to be very large